configure

```diff
@@ -984,12 +984,22 @@ for buildtype in $buildtypes; do
   # For serial and mpi versions, diagonalizers from lapack/mkl may be used
   if [ "$buildtype" = 'serial' -o "$buildtype" = 'mpi' ]; then
     if [ "$lapack" = 'yes' ]; then
-      fort_flags="$fort_flags -DLAPACK"
-      ldflags="$fort_flags -lopenblas"
-    elif [ "$mkl" = 'yes' ]; then
-      fort_flags="$fort_flags -DMKL"
-      ldflags="$fort_flags -L$INTELHOME -lmkl_intel_lp64 -lmkl_intel_thread -lmkl_core -liomp5 -lpthread"
-    fi
+		if [[ ! -n $EBROOTOPENBLAS ]]; then
+			echo "OPENBLAS is not loaded, use conventional diagnolization"
+		else
+			echo "OPENBLAS is loaded. Use LAPACK acceleration of diagnolization"
```

Comment:
echo "OPENBLAS is loaded. Diagonalizer from LAPACK will be used."

Reply:
Fixed